cli/main.go

```diff
@@ -916,6 +926,11 @@ func DefineInputs(seed *objects.Seed) ([]string, error) {
 
 		// Expand input VALUE
 		val = GetFullPath(val)
+		info, err := os.Stat(val)
+		if os.IsNotExist(err) {
+			fmt.Fprintf(os.Stderr, "ERROR: Input file %s not found\n", val)
+		}
+		size += (1.0 * float64(info.Size())) / (1024.0 * 1024.0)
```

Comment:
This is fairly self evident, but I think a comment would be justified here or in func comment to indicate sizes will be in MiBs